service/frontend/operator_handler.go

```diff
@@ -519,20 +504,9 @@ func (h *OperatorHandlerImpl) listSearchAttributesElasticsearch(
 	indexName string,
 	searchAttributes searchattribute.NameTypeMap,
 ) (*operatorservice.ListSearchAttributesResponse, error) {
-	var storageSchema map[string]string
-	if h.esClient != nil {
-		var err error
-		storageSchema, err = h.esClient.GetMapping(ctx, indexName)
-		if err != nil {
-			return nil, serviceerror.NewUnavailablef(
-				"unable to get mapping from Elasticsearch: %v", err,
-			)
-		}
-	}
 	return &operatorservice.ListSearchAttributesResponse{
 		CustomAttributes: searchAttributes.Custom(),
 		SystemAttributes: searchAttributes.System(),
-		StorageSchema:    storageSchema,
```

Comment:
Same question here. Are there uses of this and do we need backwards compatibility?

Do we need to document this in version changes?

Reply:
Ditto response